Pull request number: 1

Zven94 opened the pull request:
### Pull request number: 1

In this pull request we add the following files to our proyect.

- > Files:
- index.html
- style.css
- .gitignore
- linter.yml
- test.md
- node_modules

Zven94 committed:
Update README.md

Add new info:
- Table of Content
- Info About web
- Contact of the Dev

Zven94 committed:
Add linters.yml for html-css

I created a worflows folder into .github to store the linters.yml

Zven94 committed:
Create .gitignore

Add node_modules to .gitignore file.

Zven94 committed:
Create index.html

Created index.html with and h1 and attached to style.css

Zven94 committed:
Creat style.css

Created style.css with one class with the style of the H1

Zven94 committed:
Adding class to H1

H1 now has a class called "header-1"

Zven94 committed:
Create test.md

This file should be ignored by git

Zven94 committed:
Update .gitignore

Add test.md to the .gitignore

Zven94 committed:
Delete test.md

Delete text.md

Zven94 committed:
Create test.md

This file should be ignored by git

Zven94 committed:
Delete test.md

Deleting the file because is not ignore by .gitignore

Zven94 committed:
Create test.md

Adding a test.md that sould be ignore by .gitignore

taldr27 commented on README.md:
```diff
@@ -1 +1,201 @@
+    <li><a href="https://github.com/Zven94/Hello_Microverse.git">GitHub</a></li>
+  </ul>
+</details>
+<!--
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+-->
+<!-- Features -->
```
- [ ] Great work on using the Microverse's template for your README! However, I noticed that you commented this section please edit it to use the technologies you used in this project like HTML, and CSS3 to further enhance the professionalism of your README. 🎯

Zven94 replied:
Hello @taldr27. Thanks for your time. I'm working on all the changes you suggested. I appreciate your time, thanks for the corrections.

taldr27 replied:
You're welcome! I'm happy to help.

taldr27 commented on README.md:
```diff
@@ -1 +1,201 @@
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```
- [ ] To improve the professionalism of your `README` file, I suggest removing the commented description of all the sections and replacing them with your personalized content. This will make your `README` file more concise and easier to read. 💯

taldr27 commented on README.md:
```diff
@@ -1 +1,201 @@
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
- [ ] To enhance the professionalism of your `README` file and repository, I suggest adding a `LICENSE` file to your repository.:dart:
  _You can learn how to do it by reading **[this useful link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository)**._

taldr27 commented on test.md:
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
- [ ] Great work adding the `.gitignore` file to your project and ignoring the `test.md` file. However, following the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) it seems like you created and pushed the `test.md` file before adding the .gitignore file, which is why it's still visible in your repository. To resolve this issue, please delete the `test.md` file and push your changes to your repository again. This should ensure that the file is properly ignored and not pushed to the repository. 💪

taldr27 reviewed (changes requested):
## Hello @Zven94 , 

Good job so far! 
You're making good progress on the project, but some issues still need to be addressed before you can move on to the next stage. Keep up the good work! 💪

![almost-there](https://media.giphy.com/media/l1JojjVanlckKIHMQ/giphy.gif)

## Highlights 🎉
✅ Professional and customized README. 
✅ Linters passing.
✅ Correct use of GitHub Flow.

## Required Changes ♻️
_Check the comments under the review._

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). <img align="right" src="https://media.giphy.com/media/kReKcfrs1YoTmt2AQt/giphy.gif" alt="gif"  width="3%"/>_

Zven94 committed:
Create LICENSE

Added a MIT LICENSE for the Project

Zven94 committed:
Update README.md

- Deleted unnecessary comments 
- Improve on Tech Stack description
- Add the LICENSE

Zven94 committed:
Delete test.md

File deleted:
-test.md

AdnanOlarmmi reviewed (approved):
Hi @Zven94,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![Approved office man](https://user-images.githubusercontent.com/41163571/219113922-5007d6bb-ff59-4916-b6d3-e5aad3cb4051.gif)

## Highlights
- Nice work ignoring appropriate files.
- Professional README.

## Optional suggestions
- [ ] This is a very crucial part of this milestone, but I'm making it optional only and only because the previous reviewer mistakenly omitted it, Kindly fix it before moving forward, because it attracts invalidation in your next milestone, *Please complete all the linters configuration*.

https://user-images.githubusercontent.com/41163571/234688951-56c61ccb-66ed-456e-9f8d-827e32bd09e2.mp4


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
